.github/workflows/ci.yml

```diff
@@ -0,0 +1,118 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: [openjdk@1.8]
+        scala: [2.11.12, 2.12.10, 2.13.1]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+
+      - name: Set up Scala
+        uses: olafurpg/setup-scala@v7
+        with:
+          java-version: ${{ matrix.jdk }}
+
+      - name: Configure SBT
+        run: echo '-Dsbt.color=true -Dsbt.supershell=false' > .sbtopts
```

Comment:
I think that you might need to put these options on different lines for sbt to pick them up.

Reply:
Nope, it didn't work 🙁 I don't even know if GitHub Actions support ANSI color codes at all. They already seem to highlight lines in different shades of grey, so maybe they have their own logic for it?

Reply:
They clearly mention colors in https://github.com/features/actions, so the problem must be anywhere else.